### DISCUSSION TITLE
Font awesome icons issue

### DISCUSSION THREAD

**Discussion by 6-ril** [kimai/kimai#5214] [On-Premise: Installation & Updates]
### Describe the issue

Hello,

I have updated Kemai the last week to the version (2.24.0) and it still doesn't show font icons.
It is not relative to this version, as I checked before updating and the font icons wasn't shown.
Before asking anything, I first think that updating may fix things.
Today I see that an update is available.
I'll install it and see if this fix my issue.

It looks like this :
![image](https://github.com/user-attachments/assets/75fc65b2-7c68-4181-9c91-fabd2dce7b9d)

I found some explanation, but I'm not sure to understand
https://stackoverflow.com/questions/51097817/font-awesome-icons-no-longer-working-in-f-content-values
https://www.webdevsplanet.com/post/fontawesome-icons-not-showing

It seems to be relative to the server
I tried with Firefox and edge, it's doing the same.

The server is running Ubuntu 22.04, I'm using http only as it's an internal server.
I'm not sure what other information can help to debug ...

Any idea ?

Thanks,
Cyril

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.2.24

### How do you run Kimai?

Virtual Server or alike

### Which PHP version are you using?

8.1

**Comment by kevinpapst** [comment]
Check your browser console / network tab.
Something is blocking the font from loading

**Reply by 6-ril** [reply]
Do you mean this : 
```
GET
http://time-fr.lisam.local/build/fonts/fa-solid-900.4d986b00.woff2
[HTTP/1.1 404 Not Found 38ms]

GET
http://time-fr.lisam.local/build/fonts/fa-regular-400.21cb8f55.woff2
[HTTP/1.1 404 Not Found 32ms]

downloadable font: download failed (font-family: "Font Awesome 6 Free" style:normal weight:400 stretch:100 src index:0): status=2147746065 source: http://time-fr.lisam.local/build/fonts/fa-regular-400.21cb8f55.woff2
downloadable font: download failed (font-family: "Font Awesome 6 Free" style:normal weight:900 stretch:100 src index:0): status=2147746065 source: http://time-fr.lisam.local/build/fonts/fa-solid-900.4d986b00.woff2 


```

I found another issue (sorry I didn't check properly) where you say that Kimai doesn't support subdomain.
I think I'm using subdomain : http://time-fr.lisam.local

**Reply by kevinpapst** [reply]
Subdomain is perfectly fine.  
Check if the file exists or if something is incorrectly installed (e.g. with `git status`).
This is an environment issue or a broken installation, but not a Kimai bug.

**Reply by 6-ril** [reply]
Ok thanks a lot, I'll try that and let you know.

**Reply by 6-ril** [reply]
You're right, there's something wrong with git 

```
HEAD detached at 2.26.0
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        deleted:    public/build/fonts/fa-brands-400.455ea818.woff2
        deleted:    public/build/fonts/fa-brands-400.60127e35.ttf
        deleted:    public/build/fonts/fa-regular-400.21cb8f55.woff2
        deleted:    public/build/fonts/fa-regular-400.eb91f7b9.ttf
        deleted:    public/build/fonts/fa-solid-900.4d986b00.woff2
        deleted:    public/build/fonts/fa-solid-900.bacd5de6.ttf
        modified:   src/WorkingTime/Calculator/WorkingTimeCalculatorDay.php
        modified:   tests/WorkingTime/Calculator/WorkingTimeCalculatorDayTest.php

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        public/avatars/
        public/build/fonts/fa-brands-400.498f4d22.woff2
        public/build/fonts/fa-brands-400.4c05cd9b.ttf
        public/build/fonts/fa-regular-400.83824cd5.woff2
        public/build/fonts/fa-regular-400.96e19e1a.ttf
        public/build/fonts/fa-solid-900.4cbe3db2.woff2
        public/build/fonts/fa-solid-900.bb5be820.ttf

no changes added to commit (use "git add" and/or "git commit -a")
```

**Reply by kevinpapst** [reply]
https://www.kimai.org/documentation/updates.html#changed-files

**Comment by 6-ril** [comment]
Damned, I didn't wait for your answer.
I run :
 ```
sudo git commit -a
 sudo git add public/avatars/
 sudo git add public/build/fonts/
```
I checked the status with
`sudo git status`
and this give me : 
```
HEAD detached at 2.26.0
nothing to commit, working tree clean

```
I'm not sure to understand what does that mean, if this is an issue.

Oh !! Icons are back !!
Great
